### ISSUE TITLE
Which defconfig from arm64?

### ISSUE THREAD

**Issue by AmniX** [realme-kernel-opensource/realmeX2Pro-kernel-source#2]
There are 3 defconfigs, One is defconfig, debug and ranchu.
ranchu means emulator?
do suggest

**Comment by GT020** [comment]
> There are 3 defconfigs, One is defconfig, debug and ranchu.
> ranchu means emulator?
> do suggest

better pull from device

**Comment by mKenfenheuer** [comment]
Yes, pull the config from your device, ist located at /Proc/config.gz. However you can also Copy the defconfig from my Dirk, this ist the one i pulled from my device.
